src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -375,14 +393,16 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
                     ImmutableArray.Create(explicitlyImplementedProperty);
 
             // get-only auto property should not override settable properties
-            if (_isAutoProperty && (object)_setMethod == null && !this.IsReadOnly)
+            if ((_propertyFlags & Flags.IsAutoProperty) != 0)
```

Comment:
Is there a reason to avoid calling IsAutoProperty here? Perhaps it's not as clear because _propertyFlags is still writable in this method?

Reply:
I'm wary about changing semantics.  That said, in this case, because IsAutoProp is just a non-virtual prop that is equivalent to this, i'd be amenable to changing it.  Up to you guys :)